auth0/resource_auth0_organization.go

```diff
@@ -61,6 +61,9 @@ func newOrganization() *schema.Resource {
 			"connections": {
 				Type:     schema.TypeSet,
 				Optional: true,
+				Computed: true,
```

Comment:
Sadly we need to mark this now as computed, otherwise if we use the newly introduced `auth0_organization_connection` to add a new connection, when terraform performs a refresh the `auth0_organization` resource will see that a new connection was added "out-of-bound" and it will try to remove it, but the new resource will try to add it again, causing an infinite loop. 

We're marking this however as well as deprecated and it will get removed in the future, once everyone ports over to the new way of adding and removing connections on orgs, through the new resource.